bioblend/_tests/test_util.py

```diff
@@ -31,9 +36,40 @@ def skip_unless_galaxy(min_release=None):
         galaxy_release = os.environ.get('GALAXY_VERSION', None)
         if galaxy_release is not None and galaxy_release.startswith('release_') and galaxy_release < min_release:
             return unittest.skip(OLD_GALAXY_RELEASE % (galaxy_release, min_release))
-    for prop in ['BIOBLEND_GALAXY_URL', 'BIOBLEND_GALAXY_API_KEY']:
-        if prop not in os.environ:
-            return unittest.skip(NO_GALAXY_MESSAGE)
+
+    if 'BIOBLEND_GALAXY_URL' not in os.environ:
+        return unittest.skip(NO_GALAXY_MESSAGE)
+
+    if 'BIOBLEND_GALAXY_API_KEY' not in os.environ and 'BIOBLEND_GALAXY_MASTER_API_KEY' in os.environ:
+        galaxy_url = os.environ['BIOBLEND_GALAXY_URL']
+        galaxy_master_api_key = os.environ['BIOBLEND_GALAXY_MASTER_API_KEY']
+        gi = bioblend.galaxy.GalaxyInstance(galaxy_url, galaxy_master_api_key)
+
+        if 'BIOBLEND_GALAXY_USER' in os.environ:
```

Comment:
I think it would be better to check for `GALAXY_USER_EMAIL` here (and skip otherwise), because that needs to be mentioned in the `admin_users` line of `config/galaxy.ini` for some tests.